app/presenters/document/field_group/section_presenter.rb

```diff
@@ -0,0 +1,204 @@
+# frozen_string_literal: true
+
+module Document
+  ##
+  # Presenter for a record metadata "section", which may gather data from multiple
+  # source fields
+  module FieldGroup
+    class SectionPresenter < ApplicationPresenter
+      attr_reader :document, :controller, :group, :definition
+
+      delegate :search_field, :entity, :fields, :exclude_vals, :max, :title,
+               :ga_data, :map_values, :format_date, to: :definition
+
+      # @param document [Europeana::Blacklight::Document]
+      # @param controller [ActionController::Base]
+      # @param group [Document::FieldGroupPresenter]
+      # @param definition [OpenStruct]
+      def initialize(document, controller, group, definition)
```

Comment:
this is fine, but is it worth wile trying to align the flow with the initializer for the FieldGroupPresenter?
So loading the value presenters into @value_presenters similarly or how the FieldGroupPresenter initializes with the definitions for it's sections.

Reply:
I am not too sure what you mean here. Could you show some code to illustrate?

Reply:
the FieldGroupPresenter loads in section presenters for each section at initialization:
```
    ...
    @sections = @definition[:sections].map do |section_definition|
      Document::FieldGroup::SectionPresenter.new(document, controller, self, OpenStruct.new(section_definition).freeze)
    end
end
```

the sections don't load in the value_presenters until .display is called
so for consistency in the initializer here we could do something like: 
```
@value_presenters = value_presenters
```
however the logic for determining the value presenters is a bit more complicated for the section presenters than the simple mapping we can do in the FieldgroupPresenter initializer. Therfore I was asking whether it's worth trying to move this into the initializer.

Reply:
or potentially move the assignment of @sections out of the initializer in the FieldGroupPresenter class.

Reply:
Understood. I have moved the initialisation of the section presenters out of the field group presenter initializer, in 32cf610c20527a1fd66e626e4670eab3a02ea6e6.